src/main/java/io/jenkins/plugins/ml/IPythonBuilder.java

```diff
@@ -61,17 +58,11 @@ public String getCode() {
         return code;
     }
 
-    public static InterpreterManager getIPythonInterpreterManager(IPythonUserConfig jobUserConfig) {
-        IPythonInterpreterManager interpreterManager = configBasedManagerMap.get(jobUserConfig);
-
-        if(interpreterManager == null ){
-            interpreterManager = new IPythonInterpreterManager(jobUserConfig);
+    public InterpreterManager getIPythonInterpreterManager(IPythonUserConfig jobUserConfig) throws InterpreterException {
+            InterpreterManager interpreterManager = new IPythonInterpreterManager(jobUserConfig);
```

Comment:
So now we get our interpreterManager without having to call the Hash Map , this looks good

Reply:
yes, shivay. 
cheers :)